keycloakSetup/docker-compose.yaml

```diff
@@ -0,0 +1,17 @@
+version: '3.7'
+
```

Comment:
Maybe we want to keep this docker-compose file in the box folder and remove it from here?

Reply:
Okay... done...